docs/src/solutions.md

```diff
@@ -17,6 +17,7 @@ optimization step. Typical questions include:
  - Do I have a solution to my problem?
  - Is it optimal?
  - Do I have a dual solution?
+ - How sensitive is the solution to data uncertainty?
```

Comment:
I would call this "data perturbations" instead of "data uncertainty". Uncertainty is an overloaded term and makes me think of robust or stochastic programming.